chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -37,6 +37,7 @@ general.notNow                     = Not Now
 general.passed						= Passed
 general.failed						= Failed
 general.and							= and
+general.andJoiner					= \u2068%S\u2069\u2066 and \u2069\u2068%S\u2069
```

Comment:
Control characters make sure an RTL name won't put the entire string in RTL mode in a `dir="auto"` element.

Reply:
Can you add a comment with the characters being used here and what they do?